tests/integration/SdkTest_test.h

```diff
@@ -62,6 +64,50 @@ class MegaLoggerSDK : public MegaLogger {
     void log(const char *time, int loglevel, const char *source, const char *message);
 };
 
+struct TransferTracker : public ::mega::MegaTransferListener
+{
+    std::atomic<bool> started = false;
```

Comment:
It seems that Jenkins complains about this initialization. It happened to me as well in my first usage of `atomic_bool` and I solved it by using the initialization list:
```suggestion
    std::atomic<bool> started = {false};
```